ClosedXML/Excel/ConditionalFormats/Save/XLCFContainsConverter.cs

```diff
@@ -11,8 +8,11 @@ internal class XLCFContainsConverter : IXLCFConverter
         public ConditionalFormattingRule Convert(IXLConditionalFormat cf, int priority, XLWorkbook.SaveContext context)
         {
             String val = cf.Values[1].Value;
-            var conditionalFormattingRule = new ConditionalFormattingRule { FormatId = (UInt32)context.DifferentialFormats[cf.Style], Operator = ConditionalFormattingOperatorValues.ContainsText, Text = val, Type = cf.ConditionalFormatType.ToOpenXml(), Priority = priority };
-
+            var conditionalFormattingRule = XLCFBaseConverter.Convert(cf, priority);
```

Comment:
In many places, you changed the style of initialisation, like here. Why did you do that? I don't think it's relevant to this change.

Reply:
I did it because then i only have to set StopIfTrue in one place from the Cf that we read in, which would be in the base constructor. I believe there are other properties that are currently unsupported, and should someone ever have to implement them, they would only have to make the code change in one spot: http://www.datypic.com/sc/ooxml/e-ssml_cfRule-1.html

Reply:
Ah ok. I see now. It's fine.